tests/Behat/Context/Api/Admin/ManagingChannelsContext.php

```diff
@@ -147,6 +151,37 @@ public function iSpecifyDaysAsTheLowestPriceForDiscountedProductsCheckingPeriod(
         $this->client->addRequestData('lowestPriceForDiscountedProductsCheckingPeriod', $days);
     }
 
+    /**
+     * @When I exclude the :taxonName taxon from showing the lowest price of discounted products
+     * @When I exclude the :firstTaxon and :secondTaxon taxons from showing the lowest price of discounted products
+     */
+    public function iExcludeTheTaxonFromShowingTheLowestPriceOfDiscountedProducts(string ...$taxonNames): void
+    {
+        $taxons = [];
+        foreach ($taxonNames as $taxonName) {
+            $taxons[] = $this->iriConverter->getIriFromItem($this->taxonContext->getTaxonByName($taxonName));
```

Comment:
IMO you can create a new transformer and put this logic there to get an array of taxons instead of taxons names